README.md

```diff
@@ -1,13 +1,19 @@
-# Network Explorer #
+# OpenTelemetry eBPF #
 
-Network Explorer kernel collector is an agent that can collect low level telemetry
-straight from the Linux Kernel using the [eBPF technology](https://ebpf.io/).
-It does so with negligible overhead towards compute and network resources.
+The OpenTelemetry eBPF project develops components that collect and analyze
+telemetry from the operating system, cloud, and orchestrators. Its initial focus
+is on collecting network data to enable users to gain insight into their distributed 
+applications.
 
-This telemetry is then sent to a pipeline that can enrich it and provide
-invaluable insight about you distributed application.
+The _kernel collector_ gathers low level telemetry straight from the Linux
+Kernel using the [eBPF](https://ebpf.io/). It does so with negligible compute and 
```

Comment:
"kernel (lowercase) using the eBPF technology"?

Reply:
changed Kernel->kernel

for eBPF, it is becoming more common to just call it eBPF. Instead of "using the eBPF" -> "using the eBPF technology", did ->"using eBPF", hope that's ok.